src/inputs/Table/Table.js

```diff
@@ -12,10 +12,8 @@ import { DateUtils } from '@cosmotech/core';
 import { getColumnTypes, getDefaultColumnsProperties } from './ColumnTypes.js';
 import { TABLE_DATA_STATUS } from './TableDataStatus';
 import { ErrorsPanel, TooltipInfo } from '../../misc';
-import { getCommonInputStyles } from '../style';
```

Comment:
Your commit removes all recent development for the common component styles and the`isDirty` prop handling, you probably missed something during a merge or rebase